app/models/katello/content_view.rb

```diff
@@ -469,7 +469,7 @@ def check_remove_from_environment!(env)
       }
 
       dependencies.each do |key, name|
-        if (models = self.association(key).scoped.in_environment(env)).any?
+        if (models = self.association(key).scope.in_environment(env)).any?
```

Comment:
This one is a bit baffling to me. I've never seen `association` used before and [it's not documented unfortunately](http://git.io/v0tsb). It looks like it's a bit different than just calling the association as `where`, `all`, etc are defined for what association returns:

```
>> u.roles.class
=> Role::ActiveRecord_Associations_CollectionProxy
>> u.roles.all
=> #<ActiveRecord::AssociationRelation [...]>
>> u.association(:roles).class
=> ActiveRecord::Associations::HasManyAssociation
>> u.association(:roles).where(nil)
undefined method `where' for #<ActiveRecord::Associations::HasManyAssociation:0x007fb96047e188>
>> u.association(:roles).all
undefined method `all' for #<ActiveRecord::Associations::HasManyAssociation:0x007fb96047e188>
```